SiVL19/coco_evaluation.py

```diff
@@ -0,0 +1,79 @@
+import re
+import argparse
+import pandas as pd
+from pycocoevalcap.bleu.bleu import Bleu
+from pycocoevalcap.meteor.meteor import Meteor
+from pycocoevalcap.rouge.rouge import Rouge
+
```

Comment:
Some comments can be added here if you made any changes or about where did you take this file from.